README.md

```diff
@@ -122,40 +140,40 @@ to interact with Coq.
 Towards this goal, `coq-lsp` extends and will eventually replace `coq-serapi`,
 which has been used by many to that purpose.
 
-## Planned features
+### Planned features
```

Comment:
I would perhaps start each entry here with "We plan to" to make it more obvious that it is not the case today.

Reply:
Yes, moved out of the way, this is still very drafty and not worthy of main page space.